manifests/charts/install-Ambient.md

```diff
@@ -0,0 +1,89 @@
+# Istio Ambient Installation with Helm
```

Comment:
@bleggett could you please review this doc, and when it's good I'll update the installation page in the doc site.

Reply:
@hanxiaop Rather than put this here in-repo, can you PR the update to the actual published Ambient quickstart here? The contents are fine, but it should go in the main docs repo.

https://github.com/istio/istio.io/blob/master/content/en/docs/ops/ambient/getting-started/index.md